packages/components/segmented-control/src/segmented-control.tsx

```diff
@@ -421,7 +447,7 @@ export const SegmentedControlButton = forwardRef<
     } = useSegmentedControl()
 
     const { index, register } = useDescendant({
-      disabled: isDisabled || isReadOnly,
+      disabled: disabled || readOnly,
     })
```

Comment:
@nathanlao @hirotomoyamada 
I think @nathanlao's modifications were correct, but after looking at the [Tab PR](https://github.com/yamada-ui/yamada-ui/pull/3527/files#diff-797ad61210a8f774540381a3d271654ebf269e80aaad835b3afe97d6af2b4957), I'm wondering if I might have been wrong in my review.

In the `Tab`modification, `isDisabled` and `isFocusable` are not destructured, and only the renamed `disabled` and `focusable` are passed as arguments to `useTabDescendant`.
I was concerned that when `isDisabled` or `isFocusable` props are passed to `Tab`, these options wouldn't be properly registered as `disabled` options in the `register` argument. However, this PR has been approved.

In light of this, the correct revision might be the following:
```ts
export const SegmentedControlButton = forwardRef<
  SegmentedControlButtonProps,
  "input"
>(
  (
    {
      className,
      children,
      disabled,
      readOnly,
      value,
      motionProps,
      onChange,
      ...rest
    },
    ref,
  ) => {
    const [, mounted] = useMounted({ rerender: true })
    const {
      styles,
      value: selectedValue,
      getInputProps,
      getLabelProps,
    } = useSegmentedControl()

    const { index, register } = useDescendant({
      disabled: disabled || readOnly,
    })
```
I considered it but couldn't figure it out, so please let me know which is correct🙇‍♀️

Reply:
@hirotomoyamada 
I would also want confirmation from Mr. Yamada as well.

Reply:
@nathanlao @odendayoko 
I think @nathanlao 's modifiers are correct and `Tab`'s modifiers are not. With `isDisabled` props, I could not confirm the behavior of skipping disabled tabs and moving on to the next tab in the documentation (with `disabled` props ok)

The Tab component seemed to be implemented correctly on the face of it, but I think that is due to use-clickable.

Reply:
I've created an Issue.(#3797 )
If I'm wrong, tell me please!

Reply:
@nathanlao 

You fit the bill. Congratulations.